test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -200,7 +200,7 @@ def test_smmodelparallel_gpt2_multigpu_singlenode(ecr_image, instance_type, sage
     framework, framework_version = get_framework_and_version_from_tag(ecr_image)
     if framework == "pytorch" and Version(framework_version) in SpecifierSet("==1.9.*"):
         pytest.skip("Skipping the test for PT1.9")
-    instance_type = "ml.p3.16xlarge"
+    instance_type = "ml.p4de.24xlarge"
```

Comment:
Can we change to `ml.p4d.24xlarge` instead, as `p4de` is new, and not widely available yet, based on offline discussion with our team, thanks.

Reply:
Will do

Reply:
Does the test needs to be ran for `p3.16xlarge` as well? Do we miss any coverage for existing functionality if we remove the `p3.16xlarge` instance type? If so, please parameterize `instance_type`  with both the instance types for pytest to pickup the configuration.

Reply:
Thanks for bringing it up, our team confirmed that `p4d` should be sufficient.